cmd/minishift/cmd/start_test.go

```diff
@@ -413,8 +413,11 @@ func Test_getslice_withCommandLine(t *testing.T) {
 	viper.Set("foo", []string{"hello", "world"})
 	defer viper.Reset()
 
-	if getSlice("foo") != nil {
-		t.Errorf("expected %+v. Got %+v", nil, getSlice("foo"))
+	expectedSlice := []string{"hello", "world"}
+	for i, v := range getSlice("bar") {
```

Comment:
Same btw for _Test_getslice_withConfig_. You need to assert the size, otherwise it will be easy to break the test unnoticed.